completions/makefile.completion.sh

```diff
@@ -1,3 +1,3 @@
 # Add completion for Makefile
 # see http://stackoverflow.com/a/38415982/1472048
-complete -W "\`grep -oE '^[a-zA-Z0-9_-]+:([^=]|$)' Makefile | sed 's/[^a-zA-Z0-9_-]*$//'\`" make
+complete -W "\`find . -maxdepth 1 -iname '*makefile' -exec grep -oE '^[a-zA-Z0-9_-]+:([^=]|$)' {} \; | sed 's/[^a-zA-Z0-9_-]*$//'\`" make
```

Comment:
@jeansebastienh I haven't tested it, but how about the following one?

```suggestion
complete -W "\`shopt -u nullglob; shopt -s nocaseglob; grep -oE '^[a-zA-Z0-9_-]+:([^=]|$)' *makefile 2>/dev/null | sed 's/[^a-zA-Z0-9_-]*$//'\`" make
```

Reply:
It's working like a charm. 

Thanks 👍🏼

Reply:
Thank you for checking! LGTM now. Then let's merge this PR! After applying additional cosmetic changes, I'll merge it. (You might have already noticed it, but I joined the maintainer team last week so can process PRs now.)